neevaco/neeva-ios
# Show default browser interstitial for "skip to browser" user

* Show default browser interstitial to new users who skip to browser without sign in
* Add explicit skip to browser logging for analytics purpose
* Keep a new default flag to tell an user has seen the default browser interstitial via skip to browser flow so we can prompt the user again when they sign in

@darinwf, user still see the interstitial once. If they have seen the interstitial via skip to browser, they won't see it again when they sign in (but we can design for this case in the future as it uses separate flag for state)

## Checklists

### How was this tested?

### Manual test cases

New user flow via skip to browser and sign in flow. They both should see the interstitial. Look at Proxyman for logging

### Screenshots and screen recordings
Images or videos showing the effect of your PR.  Examples include:
* For new Views, add images of `_Previews` you have added for both light
  and dark mode
* Before and after screenshots showing the effect of your styling changes
* Videos showing you performing the flow that you have changed

## Issues addressed
* Link(s) to Github issue(s) (if applicable)
